core/src/main/scala/org/apache/spark/deploy/history/ApplicationCache.scala

```diff
@@ -156,18 +156,19 @@ private[history] class ApplicationCache(
    */
   @throws[NoSuchElementException]
   private def loadApplicationEntry(appId: String, attemptId: Option[String]): CacheEntry = {
-    logDebug(s"Loading application Entry $appId/$attemptId")
+    logDebug(s"Loading application Entry $appId" + attemptId.map { id => s"/$id" }.getOrElse(""))
```

Comment:
If you want to change `Some(XX)`, shall we try to use `mkString` simply?
```scala
scala> Seq(Some("attempt1"), None).foreach { attemptId => println(s"appId/$attemptId") }
appId/Some(attempt1)
appId/None

scala> Seq(Some("attempt1"), None).foreach { attemptId => println(s"appId/${attemptId.mkString}") }
appId/attempt1
appId/
```

Reply:
Thank you for your suggestion. I'm not sure if the attemptId doesn't exist, would it be more friendly to display as **appId** instead of **appId/**? Looking forward to your suggestion. 🙂